webpack.config.js

```diff
@@ -108,7 +125,11 @@ const webpackConfig = {
 		rules: [
 			{
 				test: /\.jsx?$/,
-				exclude: /node_modules[\/\\](?!notifications-panel)/,
```

Comment:
@ockham since this PR is still up in the air and requires more complex changes, could we start making steps by expanding this exclude test? for the time being why not add a whitelist of modules to bring in to our build?

```js
exclude: {
	and: {
		'node_modules',
		{
			not: [
				'node_modules/notifications-panel',
				'node_modules/other-untranspiled-library',
				…
			]
		}
	}
}
```

or with a function…

```js
exclude: path => {
	if ( ! /node_modules/.test( path ) ) {
		return false;
	}

	const whitelist = {
		'notifications-panel': true,
		'other-untranspiled-library': true,
		…
	}

	return whitelist.hasOwnProperty( path.split( '/' )[ 1 ] );
}
```

at least this way we could take advantage of those library and we wouldn't add much complexity at all to the build and it would be easy to make further refinements in the future

Reply:
Sounds good! FWIW, I think I prefer the `and`/`not` approach which seems more transparent (for this rather simple use case) than a dedicated function.